src/convolution.cpp

```diff
@@ -447,7 +447,11 @@ std::size_t ConvolutionDescriptor::ForwardGetWorkSpaceSize(Handle& handle,
         miopenConvSolution_t sol;
         GetForwardSolutions(handle, wDesc, xDesc, yDesc, 1, &count, &sol);
         if(count < 1 || (fm.IsHybrid() && sol.time < 0))
+        {
+            ctx.skip_solutions_that_take_long_time_to_build_and_have_narrow_coverage =
```

Comment:
Please shorten this variable name and preferably make this an `int` to indicate cost estimate if @atamazov agrees. If not at least rename it to something simpler.

Reply:
Cost estimate (or a build time?) is a good idea that allows for more flexibility. But we need also indication of _coverage_ to make a decision (skip or use). Also I am not sure that we need this flexibility right now. Let's keep this idea in mind for the future.

Reply:
WRT naming, see https://github.com/ROCmSoftwarePlatform/MIOpen/pull/326#issuecomment-654483612:

> ...`skip_solutions_that_take_long_time_to_build_and_have_narrow_coverage`. This ugly name tells anything (and BTW denotes [quirky architecture](https://github.com/ROCmSoftwarePlatform/MIOpen/issues/299#issuecomment-645698370)).